frontend/scripts/react-components/dashboard-element/dashboard-widget/table-modal/table/table.scss

```diff
@@ -0,0 +1,25 @@
+@import 'styles/settings';
+
+.c-table {
+  .table-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
```

Comment:
I think `space-evenly` is not supported on `flex` in Edge. Also `flex-direction: row;` is not needed. Maybe we need to use `display: grid;`? We can, because we don't support IE 😬

Reply:
Actually, we don't need this 😆